x-pack/plugins/threat_intelligence/public/modules/indicators/components/indicators_table/indicators_table.tsx

```diff
@@ -114,6 +116,10 @@ export const IndicatorsTable: VFC<IndicatorsTableProps> = ({
     },
   ];
 
+  if (!loading && !indicatorCount) {
```

Comment:
is that how we want the Empty State to show? I know in some other places in Kibana this message appears inside the skeleton of the table... maybe we need to check with @ferenrigue ?

Reply:
asked about it in public on slack:)